src/care/LoopFuser.h

```diff
@@ -1257,15 +1258,18 @@ void LoopFuser<REGISTER_COUNT, XARGS...>::registerAction(const char * fileName,
 
 #define FUSIBLE_KERNEL_PHASE(PRIORITY) FUSIBLE_KERNEL_PHASE_R(PRIORITY, CARE_DEFAULT_LOOP_FUSER_REGISTER_COUNT)
 
+// Flush the fusible actions if needed (if we have reached the buffer limit, for example).
+// All fusers are flushed asynchronously except the active observer, which is flushed synchronously
+// to prevent possible corruption of the previous loop with buffer information for the next loop.
 #define FUSIBLE_FLUSH_IF_NEEDED  if (FusedActions::flush_now) { \
                                     for ( FusedActions *__fuser__ : { \
                                        static_cast<FusedActions *> (LOOPFUSER(256)::getInstance()),\
                                        static_cast<FusedActions *> (LOOPFUSER(128)::getInstance()),\
-                                       static_cast<FusedActions *> (LOOPFUSER(64)::getInstance()),\
-                                       FUSED_ACTION_INSTANCE_32 FUSED_INSTANCE_COMMA \
```

Comment:
Did you mean to remove FUSED_ACTION_INSTANCE_32?

Reply:
I did not.  Good catch!